content/blog/build-and-release-hugo-blog-site-using-azure-pipelines.md

```diff
@@ -0,0 +1,208 @@
+  inputs:
+    destination: '$(Build.ArtifactStagingDirectory)'
+```
+
+_You can find a description of the possible task parameters [here](https://github.com/giuliov/hugo-vsts-extension/blob/master/README.md). Have a look at the [task.json](https://github.com/giuliov/hugo-vsts-extension/blob/master/hugo-task/task.json) if you need the exact input names._
+
+The last step is to publish the generated Hugo site as an artifact of our build. This will make it possible to use the result in a release pipeline when publishing the site.
+
+```yaml
+- task: PublishPipelineArtifact@0
+  displayName: 'Publish Hugo site as artifact'
+  inputs:
+    artifactName: 'hugo-site'
+    targetPath: '$(Build.ArtifactStagingDirectory)'
+```
+
+That's it. You can click 'Save and run'. Provide a comment and click 'Save and run' again. This will create an 'azure-pipelines.yml' file in your blog repository which contains your build pipeline. You can find the final azure-pipelines.yml for my blog site [here](https://github.com/ronaldbosma/blog-code-examples/blob/master/BuildAndReleaseHugoSiteUsingAzurePipelines/azure-pipelines.yml).
+
+Because of the trigger on master it will start a new build immediately. After your build succeeds it should contain an artifact as shown in the image below.  
+![Build artifacts](../../static/images/build-and-release-hugo-site-using-azure-pipelines/hugo-site-artifacts.png)
+<!-- ![Build artifacts](../../../../../images/build-and-release-hugo-site-using-azure-pipelines/hugo-site-artifacts.png) -->
+
+### Step 4: Publish Hugo site
+
+Now that we have a successful build we can create a release pipeline. It will take the generated Hugo site and publish it to GitHub Pages.
+
+_We're going to use a task to push the Hugo site to a GitHub repository. If you want to push to a local repository or have some extra requirements. Have a look at [this publish.ps1 script](https://github.com/JamesRandall/Vsts-GitHub-Pages-Publish/blob/master/buildAndReleaseTask/publish.ps1) that's used by the task._
+
+To configure the release pipeline:
+
+- Open your Azure DevOps project.
+- In the left menu choose Pipelines > Releases.
+- Click the 'New pipeline' button.
+- Select the 'Empty job' template.
+- Give the stage a name. E.g. 'GitHub Pages'.
+- Click 'Add an artifact'.
+- Select 'Build' as the source type.  
+  As the source, select the build pipeline we've just created.  
+  Enter a different source alias if you want, like 'blog'.  
+  ![Add artifact](../../static/images/build-and-release-hugo-site-using-azure-pipelines/release-add-an-artifact.png)
+  <!-- ![Add artifact](../../../../../images/build-and-release-hugo-site-using-azure-pipelines/release-add-an-artifact.png) -->
+  
+- Enable the 'Continuous deployment trigger' so the release will automatically start after the build succeeds.  
+![Continuous deployment trigger](../../static/images/build-and-release-hugo-site-using-azure-pipelines/release-continuous-deployment-trigger.png)
```

Comment:
Ik kan duidelijk zien dat die pijl met paint is gemaakt :P. Misschien kan je met draw.io een mooiere png maken en die boven deze image leggen?